src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -211,13 +227,25 @@ export const useTakeNowModal = ({ dataTestId }: UseTakeNowModalProps) => {
       }
     }, [targetSubject, sourceSubject, loggedInUser, isSelfReporting]);
 
+    /**
+     * Handle participant search. It can be a combination of team members and any participants
+     * (full account, pending, and limited), or just team members and full account participants
+     */
     const handleSearch = useCallback(
       async (
         query: string,
-        types: [SearchType, ...SearchType[]],
+        types:
+          | [AnyTeamSearchType, ...AnyTeamSearchType[]]
+          | [FullTeamSearchType, ...FullTeamSearchType[]],
       ): Promise<ParticipantDropdownOption[]> => {
         const requests = [];
 
+        // @ts-expect-error Yes, the array can in fact contain this value
```

Comment:
I'm not so sure either. For me `typeof types` initially resolves to `any`, so `(typeof types)[number][]` is just `any[]`. I'm good with leaving this as-is for now

Reply:
Hm, that's very odd, since we explicitly define the type for the `types` parameter a couple lines up, so I have no idea why `typeof types` would be `any` 🤔.

FWIW, this worked locally and provided intellisense autocompletion for the possible `.includes` values.